oauth2/google.go

```diff
@@ -64,7 +66,7 @@ func (g *Google) Config() *oauth2.Config {
 
 // PrincipalID returns the google email address of the user.
 func (g *Google) PrincipalID(provider *http.Client) (string, error) {
-	srv, err := goauth2.New(provider)
```

Comment:
This call is [deprecated](https://godoc.org/google.golang.org/api/oauth2/v2#New)